funnel/forms/account.py

```diff
@@ -343,8 +343,11 @@ def validate_verification_code(self, field):
 
 class ModeratorReportForm(forms.Form):
     report_type = forms.SelectField(
-        __("Report type"), validators=[forms.validators.DataRequired()]
+        __("Report type"), coerce=int, validators=[forms.validators.DataRequired()]
     )
 
     def set_queries(self):
-        self.report_type.choices = list(MODERATOR_REPORT_TYPE.items())
+        self.report_type.choices = [
+            (idx, report_type.title)
+            for idx, report_type in MODERATOR_REPORT_TYPE.items()
+        ]
```

Comment:
The original idea of `NameTitle` was to do this instead, to avoid exposing integer codes to the UI:

```python
self.report_type.choices = [(v.name, v.title) for v in MODERATOR_REPORT_TYPE.values()]
```

However, we don't have an API to remap from name to value. We need to revisit this.